README.md

```diff
@@ -1,24 +1,53 @@
-[![posh-git on Chocolatey](https://img.shields.io/chocolatey/dt/poshgit.svg)](https://chocolatey.org/packages/poshgit/)
+## Overview
 
 posh-git is a PowerShell module that integrates Git and PowerShell by providing Git status summary information that can be displayed in the PowerShell prompt, e.g.:
+
 ```powershell
 C:\Users\Keith\GitHub\posh-git [master ≡ +0 ~1 -0 !]>
 ```
+
 posh-git also provides tab completion support for common git commands, branch names, paths and more.
 For example, with posh-git, PowerShell can tab complete git commands like `checkout` by typing `git ch` and pressing the <kbd>tab</kbd> key.
 That will tab complete to `git checkout` and if you keep pressing <kbd>tab</kbd>, it will cycle through other command matches such as `cherry` and `cherry-pick`.
 You can also tab complete remote names and branch names e.g.: `git pull or<tab> ma<tab>` tab completes to `git pull origin master`.
 
-## Versions
+### Build status
+
+| AppVeyor (Windows) | Travis (Linux) | Travis (macOS) | Code Coverage Status |
+|--------------------|----------------|----------------|----------------------|
+| [![master build status][av-master-img]][av-master-site] | | | [![master build coverage][cc-master-img]][cc-master-site] |
+| [![develop build status][av-develop-img]][av-develop-site] | [![develop pscore build status][tv-develop-img]][tv-develop-site] | PLACEHOLDER | [![develop build coverage][cc-develop-img]][cc-develop-site] |
+
+## Supported Versions
+
+Two versions of posh-git are available to provide support for Windows PowerShell v2 up through PowerShell Core v6.
+
+### posh-git v0.x
+
+Version 0.x of posh-git continues to support Windows PowerShell versions 2, 3 and 4.
+And if you don't need ANSI escape sequence support, v0.x can be used on Windows PowerShell v5.x.
+See that version's [README][!!!PATH-TO-0x-README-HERE!!!] for installation instructions.
+
+### posh-git v1.x
+
+Version 1.x of posh-git is targeted specifically at Windows PowerShell 5.x and (cross-platform)
+PowerShell Core 6.x, both of which support writing prompt strings using [ANSI escape sequences][ansi-esc-code].
+On Windows, support for [Console Virtual Terminal Sequences][console-vt-seq] was added in Windows 10 version 1511.
+Consequently this version of posh-git introduces BREAKING changes with 0.x which including dropping support
+for Windows PowerShell version 2, 3 and 4 since these versions of PowerShell do not run on platforms that
+have ANSI escape sequence capability.
+There are a few other breaking changes, see the [CHANGELOG][./CHANGELOG.md] for details.
+
+The rest of this readme applies only to the posh-git version 1.x.
+
+### Active branches
+
+TODO: @dahlbyk Need to update for the new maintenance branch and moving v1 to master (and probably nuke develop, right?).  Maybe we integrate this info into the appropriate v0.x / v1.x sections above?
```

Comment:
I've been pondering this. What if we just "renamed" `master` to `v0`, `develop` to `v1`, and started working with a `v2` branch when the time comes? It would simplify "how do I link to v0's README?"-type questions, as well as targeting of PRs (is this change for v0 or v1?).

Reply:
I was thinking you'd create a v0.x branch off of master, merge develop back to master and nuke develop for now.  But you've got far more experience in this area than I.  I'll defer to your judgement.

Reply:
There's nothing special about `master` other than it being the repo's default branch, which we can change. I suppose it might mess with existing clones a bit if master disappeared, though.

I'll move forward with branch rearrangement as soon as v0.7.2 ships.

Reply:
https://github.com/dahlbyk/posh-git/pull/568